README.md

```diff
@@ -5,96 +5,32 @@ Please read this file before using this package
 
 ***This package is constantly updated and still in progress. So please use with caution ***
 
-(This package depends on the ikfast package. So please modify and put your ikfast.h file location in the mh5_ikfast.h file in the include folder before start building)
+(This package depends on the ikfast package. So please modify and put your own robots ikfast.cpp file location in the kinematics.h file in the include folder before start building)
 
-The package is divided into two parts. First part creates the data and second part visualizes the data. To run the programs in a terminal open rviz and open marker/markerArray depending upon your visualization [generally marker for spheres and markerArray for poses]. In a second terminal run the visualization program and in a third terminal run the data generation programs.
+__map_creator__ 
 
-For example: To visualize randomly generated positions.
+This package creates two types of maps. 1. Reachability Map, 2. Capability Map
+Both have their own functionalities. To create a reahability map,
 
-rosrun rviz rviz (add a marker. Fixed frame is /base_link)
+rosrun map_creator create_reachability_map
 
-rosrun reuleaux rand_pos_generator
+the default robot for now is motoman_mh5 and the resolution is 0.035. It will create a new folder called maps in the map_creator package and there you can find a databse file mentioning the robot name and resolution.
+For creating the capability map, the process is almost the same. You have to run
 
-rosrun reuleaux show_sphere
+rosrun map_creator create_capability_map
 
-(there is a purposely generated huge delay between messsages as the data is quite big and takes time to generate. It will be solved eventually. The publisher would not publish until it is subscribed to the subscriber)
+It will also create a new database file mentioning the robot name and resolution
 
-To visualize randomly generated orientations.
+__workspace_visualization__
 
-rosrun rviz rviz (add a markerArray. Fixed frame is /base_link)
+this package visualizes the reachability map and capability map in rviz. If everything goes right you can have new display panels in rviz with the tab of reachability map and capability map. Select whihch one you would like to visualize. (Dont run both of them at the same time, otherwise the system may hang up)
 
-rosrun reuleaux rand_oriention_generator 
+rosrun workspace_visualization load_reachability_map <map location><map_name>  
+rosrun workspace_visualization load_capability_map <map location><map_name> 
```

Comment:
There does not appear to be any build executable targets in the `workspace_visualization` package.  As a result, these ros nodes do not appear to exist and the rosrun commands fail.  Need to add build targets in the CMakeLists file or revise the instructions.